.gitignore

```diff
@@ -14,3 +14,4 @@ Gemfile.lock
 .ruby-gemset
 *.retry
 *.pyc
+user_playbooks
```

Comment:
We should go ahead and create this directory either with a `.gitkeep` file or with an example playbook.